.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+infrastructure/ @oaknational/platform-group 
```

Comment:
the @oaknational/platform-group will need to have write access to this repo for this to work

Reply:
This has been granted